envoy/common/platform.h

```diff
@@ -288,18 +288,20 @@ constexpr absl::string_view null_device_path{"/dev/null"};
 // Therefore, we decided to remove the Android check introduced here in
```

Comment:
This comment becomes obsolete now, right?  Maybe we can update with a comment about why we're disabling mmsg on Android.

Also, let's check w/ Ian, but given he suggested we shouldn't use mmsg even on server Envoy, perhaps we should also set `ENVOY_MMSG_MORE` to `0` on Linux, guarded by a runtime flag

Reply:
cc @RyanTheOptimist

Reply:
Since ENVOY_MSSG_MORE is a #define, I don't think we could flag-protect it.

Reply:
we could probably change this to a function shouldMmsg() or something which uses the macro logic for the OS and also a runtime flag, if we think it should be runtime flag guarded for server side.  The only call sites are in os_sys_calls_impl.cc

But do you think server Envoy usage should change this without a flag guard?  with a flag guard?  or don't change at all?

Reply:
I think if we're doing this for android only we don't need a flag guard as long as we're confident we're happy with the change (which we are).  I'd suggest we update the comment here for now and do server-side as a runtime guarded follow-up.
/wait